core/rpc/src/rpc_impl/build_tx.rs

```diff
@@ -883,6 +883,7 @@ impl<C: CkbRpc> MercuryRpcImpl<C> {
         })
     }
 
+    // TODO@chengxing: this function seems to be unnecessary, but it is used by DAO withdraw.
```

Comment:
@EthanYuan notice here.

Reply:
Yes. I will do refactoring in another pr,  thx for reminding.